db/structure.sql

```diff
@@ -654,7 +654,8 @@ SELECT
     NULL::integer AS group_count,
     NULL::integer AS student_count,
     NULL::timestamp without time zone AS updated_at,
-    NULL::timestamp without time zone AS created_at;
+    NULL::timestamp without time zone AS created_at,
+    NULL::timestamp without time zone AS deleted_at;
```

Comment:
random stuff from previous migrations is in this sql file. To clean it up just try running a rollback and then migrate. I think that will clean it up